README.md

```diff
@@ -1,4 +1,5 @@
-![runwasi logo](./art/logo/runwasi_logo_icon.svg)
+![runwasi logo light mode](./art/logo/runwasi_icon1.svg#gh-light-mode-only)
+![runwasi logo dark mode](./art/logo/runwasi_icon3.svg#gh-dark-mode-only)
```

Comment:
So we are not using the `runwasi_icon2.svg` and `runwasi_icon4.svg` right now? That's fine but I'm checking that they weren't forgotten. :-)

Reply:
Yup they are not being used, but I just want to save them in the art folder for later.